### ISSUE TITLE
Enable Creating Multiple TimeSeries

### ISSUE THREAD

**Issue by Alrobbertz** [HERMES-SOC/hermes_core#113]
For instruments that collect multiple sensor measurements, at different time cadences, they require multiple time axes to be present in the same `HermesData` object and CDF file. This is regulated by each of the different measurement's having unique DEPEND_0 metadata attributes that reference their respective time axes with the appropriate cadence. 

To achieve this we need to make updates to the `HermesData` container to allow for multiple time axes with different cadences, or time deltas. Currently the `HermesData` class only supports on "Epoch" variable encapsulated in the `timeseries` attribute. This needs to be extended, or another data structure added so that multiple "Epoch" variables can be added to the same data container and written to the same CDF.

**Comment by Alrobbertz** [comment]
@ehsteve How would you feel if we had something like this:

```python

class HermesData:

  def __init__(timeseries, ..., meta):
    if isinstance(timeseries, TimeSeries):
      self._timeseries = {"Epoch": timeseries.copy()}
    elif isinstance(timeseries, dict):
      self._timeseries = deepcopy(timeseries)
      
  @property
  def timeseries(self):
    if len(self._timeseries.keys()) == 1: # Only one Epoch
      return self.timeseries["Epoch"]
    else:
      return self._timeseries

```

This keeps the existing API functionality in-tact, but allows us to group all of the TimeSeries structures together for easier handling and IO.

**Comment by Alrobbertz** [comment]
Closed with #114
